lib/index.js

```diff
@@ -36,13 +38,20 @@ var method = module.exports = (obj, key, method) => {
   }
   flag.add(key);
 
-  Object.defineProperty(obj, key, {
-    writable: true,
+  var descriptor = {
     configurable: true,
     enumerable: true,
-    value: method
-  });
+  };
+  
+  if(value.get || value.set){
```

Comment:
Space after `if` and before `{` to keep styling consistent across the repo

We might use something like [prettier](https://github.com/prettier/prettier) in the future to make this automated

Reply:
@fent I think you should add the eslint config in the future